nixos/modules/services/networking/networkmanager.nix

```diff
@@ -224,21 +224,22 @@ in {
       };
 
       dispatcherScripts = mkOption {
+        example = [ {
+              source = pkgs.writeText "upHook" ''
+
```

Comment:
Almost good now! Except that `pkgs.writeText` will get executed/evaluated on manual build...
You can see yourself what happens: `nix-build nixos/release.nix -A manual.x86_64-linux` && `firefox .../share/doc/nixos/index.html`

```
networking.networkmanager.dispatcherScripts

    A list of scripts which will be executed in response to network events.

    Type: list of submodules

    Default: [ ]

    Example: [ { source = (build of upHook); type = "basic"; } ]

    Declared by:
    <nixpkgs/nixos/modules/services/networking/networkmanager.nix> 
```
This is why for any nontrivial types you have to set plaintext `literalExample` instead of `example`.